packages/components/src/components/Log/Log.scss

```diff
@@ -53,6 +53,31 @@ pre.tkn--log {
     right: 0;
   }
 
+  .bx--copy-btn.scroll-btn {
+    position: fixed;
+    width: 1.6rem;
+    height: 1.6rem;
+    min-height: 1.6rem; //to override Carbon's .bx--btn
```

Comment:
Let's set `size="sm"` on the buttons instead of overriding the styles. It comes out at 2rem (32px).

If the hit box gets too small it becomes a usability issue so let's keep them a little bigger.

Reply:
changed as requested